classes/UpgradeSelfCheck.php

```diff
@@ -29,13 +29,14 @@
 
 use Configuration;
 use ConfigurationTest;
+use Shop;
 
 class UpgradeSelfCheck
 {
     /**
      * Recommended PHP Version. If below, display a notice.
      */
-    const RECOMMENDED_PHP_VERSION = 70103;
+    const RECOMMENDED_PHP_VERSION = 70205;
```

Comment:
```suggestion
    const RECOMMENDED_PHP_VERSION = 70103;
```

This should be changed in the future for a version dedicated to PrestaShop 8. We shouldn't allow people to upgrade directly from < 1.7.7 to 8.

Reply:
Agreed